duckduckgo/BrowserServicesKit
# New Privacy Dashboard as BSK submodule - changes to adapt macOS (Phase 3)

Please review the release process for BrowserServicesKit [here](https://app.asana.com/0/1200194497630846/1200837094583426).

**Required**:

Task/Issue URL: https://app.asana.com/0/72649045549333/1203080778553137/f
iOS PR: https://github.com/duckduckgo/iOS/pull/1358
macOS PR: https://github.com/more-duckduckgo-org/macos-browser/pull/797
What kind of version bump will this require?: Major

**Description**:

Part 2.
It contains the following changes:
- port macOS-specific Privacy Dashboard functionality (cookie consent management, website permissions)

**Steps to test this PR**:
1. Verify building iOS in https://github.com/duckduckgo/iOS/pull/1358
2. Verify building macOS in https://github.com/more-duckduckgo-org/macos-browser/pull/797
3. Smoke test the Privacy Dashboard feature on both platforms


**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
